CRM/Utils/System/WordPress.php

```diff
@@ -789,7 +789,12 @@ public function createUser(&$params, $mail) {
     $creds['user_login'] = $params['cms_name'];
     $creds['user_password'] = $params['cms_pass'];
     $creds['remember'] = TRUE;
-    $user = wp_signon($creds, FALSE);
+
+    // Call wp_signon if we aren't already logged in
+    // For example, we might be creating a new user from the Contact record.
+    if (!current_user_can('create_users')) {
+      wp_signon($creds, FALSE);
+    }
 
     wp_new_user_notification($uid, $user_data['user_pass']);
```

Comment:
@mlutfy One more thing while this is getting attention - this call to `wp_new_user_notification()` needs updating. The [docs say](https://developer.wordpress.org/reference/functions/wp_new_user_notification/) that the second param has been deprecated since WordPress 4.3. The current params are:

```php
wp_new_user_notification( int $user_id, null $deprecated = null, string $notify = '' )
```

If both admin(s) _and_ the user need to be notified, then (because admin-only is the default) it should read:

```php
wp_new_user_notification($uid, null, 'both');
```